src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -23,7 +23,7 @@ public LinuxAppServiceEventGenerator(LinuxAppServiceFileLoggerFactory loggerFact
 
         public override void LogFunctionTraceEvent(LogLevel level, string subscriptionId, string appName, string functionName, string eventName,
             string source, string details, string summary, string exceptionType, string exceptionMessage,
-            string functionInvocationId, string hostInstanceId, string activityId)
+            string functionInvocationId, string hostInstanceId, string activityId, string runtimeSiteName)
```

Comment:
you can append runtimeSiteName at the end of the event for Linux dedicated. Just be aware that you would need to make an Antares change as well to show the data in Kusto.